src/Og.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\og\Og.
+ */
+
+namespace Drupal\og;
+
+use Drupal\Core\Config\Entity\ConfigEntityInterface;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * A static helper class for OG.
+ */
+class Og {
```

Comment:
Creating this now as the `isGroup` stuff can move to it. We should start moving stuff out of the `Drupal\og\Controller\OG` class, as controllers are meant to be for returning a response of build markup for a response (or form etc..) and could get confusing otherwise :)